### DISCUSSION TITLE
Where can I find Material Design Web for Figma?

### DISCUSSION THREAD

**Discussion by sachin-dabas** [material-components/material-web#5599] [Q&A] [answered]
Where can I find Material Design Web for Figma? If its not available, How can I adapt the Material design 3 for Web for Figma?

**Comment by asyncliz** [comment, marked as answer]
Here's a link to the [Material 3 Figma stickersheet](https://www.figma.com/community/file/1035203688168086460) from material.io.

**Reply by sachin-dabas** [reply]
Hi, how do i find the web components in this? It just has mobile components

**Reply by asyncliz** [reply]
Material components are the same on the web and mobile. Can you give more info on what you're looking for?

**Reply by sachin-dabas** [reply]
Hi, I am looking for Sidebars, top bars, buttons, and other web components.

**Reply by asyncliz** [reply]
This project has buttons available (in `@material/web/button/*` from [npm](https://npmjs.com/package/@material/web)). It doesn't currently have a sidebar or top app bar implementation.

Have you checked out our [quick start guide](https://github.com/material-components/material-web/blob/main/docs/quick-start.md)?

**Reply by sachin-dabas** [reply]
Thanks for the info
